src/types/itemTypes.ts

```diff
@@ -38,6 +41,11 @@ export interface ItemMetadata {
   bibId?: string
 }
 
+export interface ItemsResponse {
+  items: DiscoveryItemResult[]
+  status: 200 | 400
```

Comment:
I'm a bit wary of this because maybe it should align with HTTP codes an allow _any_ and all status codes?

Reply:
I think we would need an external package to get all available status codes without specifying themselves. I'm not totally sure it's necessary though since we manually specify what specific status code gets sent back in fetchItems.

Reply:
Then we should have a global list of status codes used in the app for consistency.

Reply:
Confirmed over slack that this can be done as a follow up so as not to further increase the scope of this already large PR. Here is the ticket for that: https://newyorkpubliclibrary.atlassian.net/browse/SCC-4172